CONTRIBUTING.md

```diff
@@ -113,6 +113,10 @@ We support development in the Eclipse and IntelliJ IDEs.
 For Eclipse, the minimum version that we support is [4.13][eclipse]. 
 For IntelliJ, the minimum version that we support is [IntelliJ 2017.2][intellij].
 
+[Docker](https://docs.docker.com/install/) is a necessary dependency for building Elasticsearch. You can run Elasticsearch without building all the artifacts with:
```

Comment:
Sorry if I'm overly pedantic about this but can we be a bit more accurate and say docker "is required for building _some_ Elasticsearch artifacts and executing certain test suites."